pkg/controllers/client.go

```diff
@@ -154,3 +157,22 @@ func getReadConfigJobLabel(c *daemon.Cluster) map[string]string {
 	labels["curve"] = c.Kind
 	return labels
 }
+
+// deleteSyncConfigDeployment delete the SyncConfigDeployment after the cluster is deployed.
```

Comment:
I think that put the func in k8sutil fold and deployment file is better

Reply:
ok, I've adjusted its directory